pkg/commands/artifact/run.go

```diff
@@ -126,7 +126,7 @@ func NewRunner(ctx context.Context, cliOptions flag.Options, opts ...RunnerOptio
 		TraceHTTP: cliOptions.TraceHTTP,
 	}))
 	// get the sub command that is being used or fallback to "trivy"
-	commandName := lo.Ternary(len(os.Args) > 1, os.Args[1], "trivy")
+	commandName := lo.TernaryF(len(os.Args) > 1, func() string { return os.Args[1] }, func() string { return "trivy" })
```

Comment:
[lo.NthOr](https://github.com/samber/lo?tab=readme-ov-file#nthor) is probably what we're looking for.
https://go.dev/play/p/kkQAFeEYPWC

```suggestion
	commandName := lo.NthOr(os.Args, 1, "trivy")
```

Reply:
Yes, I agree, it's better that way